### PR TITLE
Update taskrunner images for easier python version management

### PR DESCRIPTION
**Pull request by kbirk**

### CODE REVIEW

**Review by YohannParis — APPROVED** [review]
This is hilarious, but smart idea!

File: packages/equation_extraction/Dockerfile [hunk 1]
```diff
@@ -1,45 +1,35 @@
...
-RUN python --version
-RUN pip3 --version
+ENV JAVA_HOME=/usr/local/jdk-21
+ENV PATH="$JAVA_HOME/bin:$PATH"
 
-# Install nougat
-COPY ./packages/nougat_cpu/nougat-version.txt /nougatVersion.txt
-RUN COMMIT_SHA="$(cat /nougatVersion.txt)" && \
+# Install equation-extraction
+COPY ./packages/equation_extraction/equation-extraction-version.txt /extractionersion.txt
```
Comment (dgauldie) [inline packages/equation_extraction/Dockerfile:27]: spelling mistake `extractionersion`. you are missing a v

File: packages/equation_extraction/Dockerfile [hunk 2]
```diff
@@ -1,45 +1,35 @@
...
+RUN COMMIT_SHA="$(cat /extractionersion.txt)" && \
```
Comment (dgauldie) [inline packages/equation_extraction/Dockerfile:28]: same here. Make sure you change the name of the file as well

File: packages/equation_extraction/Dockerfile [hunk 3]
```diff
@@ -63,30 +53,35 @@ RUN mv /taskrunner/build/libs/*.jar .
 RUN mv /terarium-1.0.0-SNAPSHOT.jar /taskrunner.jar
 
 # ------------------------------------------------------------------------------
-
-# Set up the Python image with JRE
-FROM python:3.10-slim
+FROM python:3.11-slim
```
Comment (dgauldie) [inline packages/equation_extraction/Dockerfile:56]: 3.10 or 3.11???

**Review by dgauldie — CHANGES_REQUESTED** [review]
there are a few issues that need fixing

File: packages/equation_extraction/Dockerfile (hunk 3 above)
Comment (kbirk) [reply packages/equation_extraction/Dockerfile:56]: Who knows! I just bumped because why not

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.10.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```
Comment (dgauldie) [inline gradle/wrapper/gradle-wrapper.properties:3]: we need to downgrade to 8.4???

File: packages/server/src/main/resources/application.properties [hunk 5]
```diff
@@ -145,13 +145,14 @@ terarium.amazon.s3.default.credentials-id=default
...
 jsdelivr.url=https://cdn.jsdelivr.net
 mira-api.url=http://mira-epi-dkg-lb-c7b58edea41524e6.elb.us-east-1.amazonaws.com
-mit-proxy.url=https://mit-tr.dev.terarium.ai
-skema-py.url=https://skema-py.dev.terarium.ai
-skema-rs.url=https://skema-rs.dev.terarium.ai
-skema-unified.url=https://skema-unified.dev.terarium.ai
-cosmos-service.url=https://cosmos.dev.terarium.ai
-ciemss-service.url=https://pyciemss.dev.terarium.ai
-climate-data-service.url=https://climate-data.dev.terarium.ai
+mit-proxy.url=https://mit-tr.staging.terarium.ai
```
Comment (dgauldie) [inline packages/server/src/main/resources/application.properties:148]: if we are moving these to staging, have you made sure that everything now points to staging?

Comment (kbirk) [reply packages/server/src/main/resources/application.properties]: Fixed

File: gradle/wrapper/gradle-wrapper.properties (hunk 4 above)
Comment (kbirk) [reply gradle/wrapper/gradle-wrapper.properties:3]: someone else will need to push 8.10, 8.4 is all I can install

Comment (kbirk) [reply gradle/wrapper/gradle-wrapper.properties:3]: Wait did someone just fix these?

**Review by dgauldie — APPROVED** [review]

**Pull request merged**